scripts/automation/dismod/create_singularity.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+#$ -P proj_dismod
+#$ -pe multi_slot 5
+#$ -cwd
+#$ -j y
+#$ -o /share/temp/sgeoutput/adolgert/output.txt
+#$ -N SingDismod
+#$ -S /bin/bash
+
+set -x
+THREADS=5
+IMAGE=dismod_at:0.0.1
```

Comment:
This part has 0.0.1 built-in. It's a key place to automate. Not for this PR.

Reply:
:+1: :confetti_ball: